categories_map.py

```diff
@@ -1782,6 +1782,7 @@
         "DPGAnalysis/Phase2L1TNanoAOD",
         "DPGAnalysis/MuonTools",
         "PhysicsTools/NanoAOD",
+        "PhysicsTools/BPHNano",
```

Comment:
We want to keep the packages in alphabetical order, so could you change this to
```suggestion
        "PhysicsTools/BPHNano",
        "PhysicsTools/NanoAOD",
```